modules/core/core-api/src/main/java/com/enonic/xp/content/ContentVersionDateComparator.java

```diff
@@ -0,0 +1,40 @@
+package com.enonic.xp.content;
+
+import java.time.Instant;
+import java.util.Comparator;
+
+public class ContentVersionDateComparator
```

Comment:
final
Or maybe even internal of ContentVersions, because it is an implementation detail